### PR TITLE
Add more React Meetups

### PR DESCRIPTION
**Pull request by Asmit2952**
## Adding a meetup
- [x] Suggestion is not a duplicate
- [x] Suggestion is under the proper category
- [x] Link to meetup is included 
- [x] Name of meetup is included
- [x] Name and link are spelled correctly

## Adding a category
- [x] Category is not a duplicate
- [x] Category is spelled correctly

### CODE REVIEW

**Comment by Asmit2952** [comment]
Hey @jdwilkin4, are these meetups enough? These were the popular ones which I have attended virtually

**Comment by Asmit2952** [comment]
Hey @tyrelchambers @jdwilkin4 can you please give a look at this PR?

**Comment by jdwilkin4** [comment]
@Asmit2952 

The staff will be back in the office on Monday and someone will review your PR then.
Thank you for your patience. 👍

File: README.md [hunk 1]
```diff
@@ -29,3 +29,7 @@ We would love to know which meetups you think deserve to be included in our list
 - [State of React](https://www.reactjsmeetup.com/#/) - State of React is a React meetup hosted multiple times a year by This Dot Media. Join a panel of React experts, enthusiasts, and contributors and get the opportunity to ask questions live!
 - [React Robins](https://www.reactladies.com/) - React Robins is a supportive community for women and non-binary React enthusiasts with any level of programming experience to grow as technologists and meet other React developers. Their events are centered around technologies within the React ecosystem.
 - [Kansas City React](https://www.meetup.com/Kansas-City-React-Meetup/events/) - Kansas City React hosts monthly meetups for anyone interested in learning, teaching, and sharing React. They also have a private Slack channel where you can ask questions and get help from other members.
+- [React India](https://www.reactindia.io/) - React India is an international conference that provides a platform for developers to share and discuss their insights and experiences with React! It Provides developers from India an opportunity meet, listen, and network with React Developers and Open Source Contributors all around the world!
```
Comment (jdwilkin4) [inline README.md:32]:
```suggestion
- [React India](https://www.reactindia.io/) - React India is an international conference that provides a platform for developers to share and discuss their insights and experiences with React! It provides developers from India an opportunity to meet, listen, and network with React Developers and Open Source Contributors all around the world!
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Asmit2952** [comment]
> Thank you for your contribution!
> 
> Please review the requested changes. Once those changes have been implemented, then we can merge your PR into main. 👍

Done @jdwilkin4

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**